config/thisroot.sh

```diff
@@ -201,7 +201,12 @@ if [ "x${SOURCE}" = "x" ]; then
    elif [ -f ./thisroot.sh ]; then
       ROOTSYS=$(cd ..  > /dev/null; pwd); export ROOTSYS
    else
-      echo ERROR: must "cd where/root/is" before calling ". bin/thisroot.sh" for this version of "$SHELLNAME"!
+      if [ "$SHELLNAME" = "bash" ] ; then
+         echo ERROR: "$SHELLNAME" can only be used if the extdebug option is enabled (for BASH_ARGV to be defined)! # https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-BASH_005fARGV
+         echo Consider enabling extdebug, or using 'source' instead, or "cd where/root/is".
+      else
+         echo ERROR: must "cd where/root/is" before calling ". bin/thisroot.sh" for this version of "$SHELLNAME"!
```

Comment:
```suggestion
         echo ERROR: must "cd where/root/is" before calling ". thisroot.sh" for this version of "$SHELLNAME"!
```
I think the script is looking for `./thisroot.sh`, right?

Reply:
The script looks for both options, bin/thisroot.sh as well as ./thisroot.sh, but in the header of the file the first is preferred/specified I guess, as historically this was called just after 'make' within the source folder.
`alias thisroot=". bin/thisroot.sh"`

Reply:
I think it makes more sense to leave it as bin/thisroot.sh because ./thisroot.sh is just a fallback that actually changes the folder ROOTSYS=$(cd ..  > /dev/null; pwd);